modules/gapi/include/opencv2/gapi/gkernel.hpp

```diff
@@ -380,7 +382,7 @@ namespace gapi {
         // Leading 0 helps to handle case when KK is an empty list (kernels<>()).
 
         int unused[] = { 0, (pkg.include<KK>(), 0)... };
-        (void) unused;
+        cv::util::suppress_unused_warning(unused);
```

Comment:
There is `CV_UNUSED` macro in the _core_ module.

Reply:
As G-API is being decoupled from OpenCV in parallel, we'll need to have our own